enos/enos-scenario-autopilot.hcl

```diff
@@ -78,11 +90,8 @@ scenario "autopilot" {
   }
 
   step "create_vault_cluster" {
-    module = module.vault_cluster
-    depends_on = [
-      step.create_vpc,
-      step.build_vault,
```

Comment:
Out of curiosity, why are we no longer depending on `step.build_vault`?

Reply:
That's a great question.  In my earlier tests I thought I saw it fail unless I explicitly depended it, but with later versions of Enos I think the implicitly dependency when called within the step works, so I don't need to make it explicitly define depends_on anymore.  @ryancragun can confirm for us, if removing it is a good idea :) !

Reply:
Last I heard we still needed to explicitly include the `depends_on`, unless there's something I missed...

Reply:
https://github.com/hashicorp/vault/pull/17353#discussion_r988275001